regolith/tools.py

```diff
@@ -2104,39 +2105,47 @@ def google_cal_auth_flow():
         token.write(creds.to_json())
     # Save the credentials for the next run
 
-def add_to_gitlab(name):
-    """Takes a newly created event, and adds it to the user's google calendar
+def create_talk_repo(name, rc):
+    """Takes in the presentation name and creates a repo
 
     Parameters:
         name - the name/key of the event
+        rc - run control object
 
     Returns:
         None
     """
-
-    tokendir = os.path.expanduser("~/.config/regolith/tokens/gitlab_api")
-    os.makedirs(tokendir, exist_ok=True)
-    tokenfile = os.path.join(tokendir, 'token.json')
+    regorcdir = Path('~/dbs/rg-db-group/local')
+    regorcfile = regorcdir.expanduser().joinpath('regolithrc.json')
     data = {}
-    # print(tokenfile)
-    if os.path.exists(tokenfile):
-        # if it exists, grab the personal access token
-        with open(tokenfile) as fp:
-            data = json.load(fp)
-
-    private_token = data.get('headers').get('PRIVATE-TOKEN')
-    if private_token:
-        data['params']['name'] = name
-        try:
-            response = requests.post(data['url'], headers=data['headers'], params=data['params'])
-            with open(tokenfile, 'w') as f:
-                json.dump(data, f)
-            if response.status_code in {200, 201}:
-                print(f"repo {name} has been created in talks: https://gitlab.thebillingegroup.com/talks/{name}")
-        except requests.exceptions.RequestException:
-            print("ERROR: Issue with GitLab API call")
+
+    if rc.talk_repo:
+        token = rc.gitlab_private_token
+        repo_url = rc.talk_repo[0]['url']
+        param = rc.talk_repo[0]['params']
+        if token:
+            with open(regorcfile) as fp:
+                data = json.load(fp)
+            data['talk_repo'][0]['params']['name'] = name
+            try:
+                response = requests.post(repo_url, params=param, headers={'PRIVATE-TOKEN': '{}'.format(token)})
+                print(f"status_code: {response.status_code}")
```

Comment:
don't make the code too chatty.  Users generally don't like it.  What will a chemist in iowa do with this information for example.  If it is a debugging print that is ok, just remembe rto delete it before merging.